src/main/resources/static/javascript/mainApp/general.controller.js

```diff
@@ -910,7 +909,7 @@
                     $scope.getGroupingInformation();
                     $scope.syncDestArray = [];
                 }
-            });
+            }).catch(() => { $log.debug("Modal dismissed"); });
```

Comment:
I remember you do not need to use .catch for these .result.finally() calls. The .result.finally calls takes in two functions as parameters, one for on success and another for on error. So you can change this exact line to:
`}, () => $log.debug("Modal dismissed"));` or just `}, $log.debug("Modal dismissed"));`, I'm not completely sure if the second one is allowed.

Reply:
I tried result.finally() with two functions, but the second one isn't called when the modal is dismissed and the promise is rejected. It also still prints an error in console about a possible unhandled exception. 
It worked for result.then() with two functions, but then we would have to duplicate the stuff in the success function into the rejected function. Also, should I remove $log.debug stuff cause I don't think we care if the modal is dismissed or not for the success modals.

Reply:
Yes, I agree with removing $log.debug. We can go with an empty .catch() block. Other places in the general controller also use an empty catch block.